va_explorer/va_data_management/management/commands/run_rms.py

```diff
@@ -0,0 +1,167 @@
+from django.core.management.base import BaseCommand
```

Comment:
I can't quite tell if this (and the dhis_rms it references which was copied from openva_pipeline) is needed. The code seems to be almost the same as `run_dhis.py` and `dhis.py`/`dhis_rms.py` seem to be almost the same as well. I'd consider removing and only keeping `run_dhis.py` since it is the only file that properly references openva_pipeline as a package.

Reply:
Not sure either but confirmed that export works without both files, so I'm okay removing them